gateway-messages/tests/versioning/v11.rs

```diff
@@ -0,0 +1,33 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! The tests in this module check that the serialized form of messages from MGS
+//! protocol version 10 have not changed.
+//!
+//! If a test in this module fails, _do not change the test_! This means you
+//! have changed, deleted, or reordered an existing message type or enum
+//! variant, and you should revert that change. This will remain true until we
+//! bump the `version::MIN` to a value higher than 11, at which point these
+//! tests can be removed as we will stop supporting v11.
+
+use super::assert_serialized;
+use gateway_messages::MgsRequest;
+use gateway_messages::SerializedSize;
+use gateway_messages::SpResponse;
+
+#[test]
```

Comment:
Could you add a test for the new `SpError::Vpd(VpdError::*)` variants too? Should be basically the same as [the SprotError tests](https://github.com/oxidecomputer/management-gateway-service/blob/aa9bdaa814811c450c4ee163cf5a1815d3d383b3/gateway-messages/tests/versioning/v09.rs#L22-L59) I think.